packages/proto-codecs/src/codegen/osmosis/superfluid/tx.ts

```diff
@@ -2452,3 +2495,225 @@ export const MsgAddToConcentratedLiquiditySuperfluidPositionResponse = {
     };
   },
 };
+function createBaseMsgLockExistingFullRangePositionAndSFStake(): MsgLockExistingFullRangePositionAndSFStake {
+  return {
+    positionId: Long.UZERO,
+    sender: "",
+    valAddr: "",
+  };
+}
+export const MsgLockExistingFullRangePositionAndSFStake = {
+  typeUrl: "/osmosis.superfluid.MsgLockExistingFullRangePositionAndSFStake",
+  encode(
+    message: MsgLockExistingFullRangePositionAndSFStake,
+    writer: _m0.Writer = _m0.Writer.create()
+  ): _m0.Writer {
+    if (!message.positionId.isZero()) {
+      writer.uint32(8).uint64(message.positionId);
+    }
+    if (message.sender !== "") {
+      writer.uint32(18).string(message.sender);
+    }
+    if (message.valAddr !== "") {
+      writer.uint32(26).string(message.valAddr);
+    }
+    return writer;
+  },
+  decode(
+    input: _m0.Reader | Uint8Array,
+    length?: number
+  ): MsgLockExistingFullRangePositionAndSFStake {
+    const reader = input instanceof _m0.Reader ? input : new _m0.Reader(input);
+    const end = length === undefined ? reader.len : reader.pos + length;
+    const message = createBaseMsgLockExistingFullRangePositionAndSFStake();
+    while (reader.pos < end) {
+      const tag = reader.uint32();
+      switch (tag >>> 3) {
```

Comment:
could the `tag` be negative? If so, you might want to use `>>`

Reply:
This is generated 😅